src/rpc/rawtransaction.cpp

```diff
@@ -155,14 +155,12 @@ UniValue getrawtransaction(const JSONRPCRequest& request)
 
     if (!request.params[2].isNull()) {
         uint256 blockhash = ParseHashV(request.params[2], "parameter 3");
```

Comment:
nit: suggest you add a comment here to say that `ParseHashV()` will throw if called with a string which isn't a 64 length hex strong.